src/redis/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+const IORedis = require( 'ioredis' );
+
+/**
+ * Internal dependencies
+ */
+let { logger } = require( '../logger/index' );
```

Comment:
Given the user will pass a logger in line #L80 (or we'll default to `console`) we probably can remove this altogether and rely on the object passed. right?

Reply:
When I removed that line, I got `logger is undefined` errors because I wouldn't be defining the logger until the client is exported.

Would it be alright if I just:

`let logger = console;`
instead?

Reply:
I guess given you pass the logger in the constructor, it should be present in `this.logger` ?

Not sure if you need to explicitly set it in `this.logger` or it will be done automatically, but usually this is how it should work in a class.